source/simulator/helper_functions.cc

```diff
@@ -1821,6 +1821,80 @@ namespace aspect
   }
 
 
+  template <int dim>
+  void
+  Simulator<dim>::replace_outflow_boundary_ids(const unsigned int offset)
+  {
+    const QGauss<dim-1> quadrature_formula (finite_element.base_element(introspection.base_elements.temperature).degree+1);
```

Comment:
this should be the velocity element, as you evaluate velocity below

Reply:
Not sure. This is used to determine if the face has outflow when assembled for the temperature (or compositional) equation.

Reply:
Ah, disregard my comment then. There are identical by default anyway.